examples/Raytracer_Benchmark/RaytraceBenchmark.cpp

```diff
@@ -126,6 +129,30 @@ int main(int argc, char *argv[])
 
 int RaytraceBenchmarkCPU(cxx::RaytracerData_t &rtdata)
```

Comment:
This will need to become something like:
template <copcore::BackendType backend> int RaytraceBenchmarkCPU(RaytracerData_t<backend> &rtdata)
Needed to unify the launcher sequence, bit not for this PR

Reply:
We will need to check if RaytracerData_t will have to be templated by the backend or not